perception/map_based_prediction/include/map_based_prediction/map_based_prediction_node.hpp

```diff
@@ -199,6 +200,12 @@ class MapBasedPredictionNode : public rclcpp::Node
   std::vector<double> distance_set_for_no_intention_to_walk_;
   std::vector<double> timeout_set_for_no_intention_to_walk_;
 
+  // Stop watch
+  StopWatch<std::chrono::milliseconds> stop_watch_;
+
+  // PublishedTime Publisher
```

Comment:
```suggestion
```

These comments are not useful, they are stating the obvious. Could you remove all instances of these comments?